rshim.c

```diff
@@ -2132,7 +2799,13 @@ static int rshim_fs_init(struct rshim_backend *bd)
   char *argv[] = {"./bfrshim", "-f"};
   int i, rc;
 
+#ifdef HAVE_RSHIM_CUSE
+  if (cuse_init() != CUSE_ERR_NONE)
+    return -1;
+#endif
+
   for (i = 0; i < RSH_DEV_TYPES; i++) {
+#ifdef FUSE
```

Comment:
Is it a typo here?  Linux doesn't have this definition.

Reply:
Yes, this is a typo.

Reply:
should be fixed now.

Reply:
Other code looks ok to me. I'll try to verify it in Linux today.